src/qt/networkstyle.cpp

```diff
@@ -6,18 +6,20 @@
 
 #include "guiconstants.h"
 
+#include "chainparamsbase.h"
+#include "tinyformat.h"
+
 #include <QApplication>
 
 static const struct {
     const char *networkId;
     const char *appName;
     const int iconColorHueShift;
     const int iconColorSaturationReduction;
-    const char *titleAddText;
 } network_styles[] = {
-    {"main", QAPP_APP_NAME_DEFAULT, 0, 0, ""},
-    {"test", QAPP_APP_NAME_TESTNET, 70, 30, QT_TRANSLATE_NOOP("SplashScreen", "[testnet]")},
-    {"regtest", QAPP_APP_NAME_TESTNET, 160, 30, "[regtest]"}
+    {"main", QAPP_APP_NAME_DEFAULT, 0, 0},
```

Comment:
Unfortunately, since std::string is a non-literal type, these constants are initialized in chainparamsbase.cpp after network_styles here. If these constants were macros instead of static attributes there would be no problem.
Since now we sometimes do scripted diffs PRs, perhaps it's a good time to think about s/CBaseChainParams::MAIN/CHAINPARAMS_MAIN/ and then more s/"main"/CHAINPARAMS_MAIN/ but this can be done independently from this PR (before or after).